<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="MapTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>780</width>
          <height>505</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>780</width>
          <height>505</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>259</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>259</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>163</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>167</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="BackButton">
          <property name="text">
           <string>К настройкам</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CheckButton">
          <property name="text">
           <string>Проверка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NextButton">
          <property name="text">
           <string>Следующий</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
